reference/forms/types/options/rounding_mode.rst.inc

```diff
@@ -1,32 +1,38 @@
 rounding_mode
 ~~~~~~~~~~~~~
 
-**type**: ``integer`` **default**: ``NumberToLocalizedStringTransformer::ROUND_HALF_UP``
+**type**: ``integer`` **default**: ``\NumberFormatter::ROUND_HALFUP``
 
-If a submitted number needs to be rounded (based on the `scale`_
-option), you have several configurable options for that rounding. Each
-option is a constant on the :class:`Symfony\\Component\\Form\\Extension\\Core\\DataTransformer\\NumberToLocalizedStringTransformer`:
+If a submitted number needs to be rounded (based on the `scale`_ option), you
+have several configurable options for that rounding. Each option is a constant
+on the ``\NumberFormatter`` class:
```

Comment:
What about doing ``:phpclass:`NumberFormatter` `` here?

Reply:
Yes! Done in af24efab9a3b48323f0cece7eb8944d6d8687054